styles/lichess/catppuccin.user.css

```
@@ -0,0 +1,473 @@
+    }
+
+    /* Game selection menus */
+    .hooks__list td {
+      background-color: ~"@{overlay0}7F"; /* Using 8 digit hex codes to replace argb */
+    }
+    .hooks__list tr.join:hover td {
+      background-color: ~"@{accent-color}7F"; /* Using 8 digit hex codes to replace argb */
+    }
+
+    /* Topbar */
+    #topnav.hover section:hover > a,
+    #topnav section:active > a,
+    #topnav div,
+    .site-buttons .shown .toggle,
+    .site-buttons .dropdown {
+      background: @crust;
+    }
+    #topnav section > a {
+      color: @subtext0;
+    }
+    #topnav.hover section:hover > a,
+    #topnav section:active > a,
+    #topnav div a,
+    #dasher_app .selector button,
+    #dasher_app .head,
+    #dasher_app .piece .no-square,
+    #dasher_app .theme .list button,
+    #dasher_app .links a,
+    #dasher_app .links button,
+    #dasher_app .subs .sub,
+    .site-buttons .dropdown a,
+    .site-buttons .dropdown button {
+      color: @text;
+    }
+    #topnav.hover section:hover > a,
+    #topnav section:active > a,
+    #topnav div {
+      border-left-color: @accent-color;
+    }
+    signal > i {
+      background-color: @green;
+    }
+    #dasher_app .sub::before,
+    .is-green::before {
+      color: @green;
+    }
+    #challenge-app .empty {
+      background: @crust;
+    }
+
+    /* Popup skining */
+    dialog {
+      background: @crust;
+    }
+    .game-setup .optional-config,
+    .game-setup .ratings {
+      background: @base;
+    }
+    .game-setup .radio input:checked + label {
+      background: @green;
+    }
+
+    /* Text */
+    .site-buttons .link,
+    .button,
+    .input,
+    .optgroup,
+    .select,
+    .textarea,
+    .lobby__box td.name a,
+    .lobby__box__top .title,
+    .lobby__box .user-link,
+    .mini-game,
+    .mini-game:hover,
+    .text,
+    .ublog-post-card__title,
+    .lobby__counters a,
+    .lobby__streams .stream,
+    .lobby__timeline .entry a,
+    .lobby__about a,
+    .lobby__timeline,
+    .lobby__counters,
+    .tabs-horiz,
+    .lobby__streams .more,
+    .lobby__timeline .more,
+    .dialog-content,
+    button,
+    input,
+    optgroup,
+    select,
+    textarea {
+      color: @subtext0;
+    }
+    .site-buttons .link:hover,
+    .button.button-metal:hover,
+    .radio input:checked + label {
+      color: @text;
+    }
+    .site-title-nav__donate,
+    .lobby__streams .stream.highlight strong,
+    .utitle,
+    .lobby__support__text,
+    .lobby__support i::before {
+      color: @peach;
+    }
+    .mini-game__clock.clock--run,
+    .tabs-horiz span.active {
+      color: @red;
+    }
+    .tabs-horiz span.active,
+    .tabs-horiz span:hover {
+      border-color: @red;
+    }
+    a {
+      color: @blue;
+    }
+    /* Links */
+    .lobby__timeline .entry:hover a,
+    a:hover,
+    a:active,
+    a:focus {
+      color: @accent-color;
+    }
+
+    /* Homepage cards */
+    .lobby__app__content,
+    .box {
+      background-color: @mantle;
+    }
```

This should be a fix for the home screen boxes
```suggestion
    }
    .lpools>div {
    	background: fadeout(@crust, 80);
    	border-color: @surface0;
          &:hover {
              background-color: fadeout(@accent-color, 80);
          }
    }
```